src/components/List/List.stories.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import List from './List';
+import ClubCard from '../ClubCard/ClubCard';
+
+export default {
+  title: 'Components/List',
+  component: List
+};
+
+const data = [
+  {
+    coverUrl: 'https://via.placeholder.com/240x140',
+    title: 'Codingbug',
+    to: '/'
+  },
+  {
+    coverUrl: 'https://via.placeholder.com/240x140',
+    title: 'Code Club',
+    to: '/'
+  }
+];
+
+export const ClubCardList = () => (
+  <Router>
+    <List title="Clubs" Component={ClubCard} _id="1" />
+    <li>
+      <ClubCard
+        coverUrl={data[0].coverUrl}
+        title={data[0].title}
+        to={data[0].to}
+      />
+    </li>
+    <li>
+      <ClubCard
+        coverUrl={data[1].coverUrl}
+        title={data[1].title}
+        to={data[1].to}
+      />
+    </li>
+  </Router>
```

Comment:
Hey, just re=checked and this example isn't right. You don't need the ClubCard components in the `li` tags, just pass the data array as a prop to the List.

Reply:
Thanks for checking @RobTF9 . I've made the change you suggested and added some media queries: grid of 4 for desktops and 2 for tablets. All displaying correctly. Should the h1 be in the page components instead?